finagle-redis/src/main/scala/com/twitter/finagle/redis/ScriptCommands.scala

```diff
@@ -0,0 +1,147 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.util.ReplyFormat
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ChannelBuffers, ChannelBuffer}
+
+trait Scripts { self: BaseClient =>
+  val handleUnit: PartialFunction[Reply, Future[Unit]] = {
```

Comment:
feel free to ignore: I think these functions read better as just unit, long, bool, etc.  unit.orElse(long) seems just as clear as handleUnit.orElse(handleLong) but less verbose.